poreCov.nf

```diff
@@ -453,17 +466,28 @@ ${c_yellow}Inputs (choose one):${c_reset}
                     ${c_dim}[Lineage + Reports]${c_reset}
 
 ${c_yellow}Workflow control (optional)${c_reset}
-    --update        Always try to use latest pangolin & nextclade release [default: $params.update]
-    --samples       .csv input (header: Status,_id), renames barcodes (Status) by name (_id), e.g.:
-                    Status,_id
-                    barcode01,sample2011XY
-                    BC02,thirdsample_run
-    --extended      poreCov utilizes from --samples these additional headers:
-                    Submitting_Lab,Isolation_Date,Seq_Reason,Sample_Type
-    --nanopolish    use nanopolish instead of medaka for ARTIC (needs --fast5)
-                    to skip basecalling use --fastq or --fastq_pass and provide a sequencing_summary.txt in addition to --fast5
-                    e.g --nanopolish sequencing_summary.txt
-    --screen_reads  Determines the Pangolineage of each individual read (takes time)         
+    --update                 Always try to use latest pangolin & nextclade release [default: $params.update]
+    --samples                .csv input (header: Status,_id), renames barcodes (Status) by name (_id), e.g.:
+                             Status,_id
+                             barcode01,sample2011XY
+                             BC02,thirdsample_run
+    --extended               poreCov utilizes from --samples these additional headers:
+                             Submitting_Lab,Isolation_Date,Seq_Reason,Sample_Type
+    --nanopolish             use nanopolish instead of medaka for ARTIC (needs --fast5)
+                             to skip basecalling use --fastq or --fastq_pass and provide a sequencing_summary.txt in addition to --fast5
+                             e.g --nanopolish sequencing_summary.txt
+    --screen_reads           Determines the Pangolineage of each individual read (takes time)    
```

Comment:
maybe a separate category for the whole screen reads part so its easier to read in the --help?